### ISSUE TITLE
During add account screen reader does not read the dialog title

### ISSUE THREAD

**Issue by test-account8-live-com** [AzureAD/azure-activedirectory-library-for-dotnet#1099]
When ADAL pops up the sign in dialog, narrator does not announce that a dialog has popped up and starts reading the content of the dialog. It is an A11yMAS violation.

NOTE:
Same issue does not exists with NVDA.

MAS Reference:
MAS 4.2.1 - Object Information (33A) - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7ba12f3779-bdf1-40c2-bbcc-0e314ea94858%7d

**Comment by atiqhaqu** [comment]
Sorry I did not realize I was logged in to GitHub with a test account.

**Comment by jmprieur** [comment]
triage: accessibility issue
cc;  @henrik-me 

**Repro steps:**:

- Run one of the [.NET desktop samples](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-code-samples#desktop-and-mobile-public-client-applications-calling-microsoft-graph-or-a-web-api) (use the app creation script to run quicker) for instance: [Dotnet-native-desktop](https://github.com/Azure-Samples/active-directory-dotnet-native-desktop)
- Win-R narrator
- sign-in in the application

**Expected**
the narrator reads the dialog title that pops-up : "sign-in to your account"

**Actual**
the dialog title is not read (the content of the page is provided by Azure AD).
The ADAL.NET dialog containing the embbeded web view is not accessible

**Comment by jmprieur** [comment]
After going through the repro, setting this as a P2 as it says "login.onmicrosoftone.com sign-in with email, phone, skype" … not ideal, but not completely wrong? @atiqhaqu what's your opinion
cc: @henrik-me @bgavrilMS

**Comment by henrik-me** [comment]
@jmprieur : not really sure what is expected here. I suppose we need to validate all our samples with narrator and other tools for accessibility, however not sure what we are actually expected to do differently.

**Comment by henrik-me** [comment]
@atiqhaqu : what is the actual test scenario you were running? What @jmprieur mentions or?

**Comment by atiqhaqu** [comment]
@jmprieur the issue here is the narrator does not read the dialog title and there is no way to know using narrator that a new dialog has popped up. This shows up in our sign in/add account flow in VS.

For priority, this bug is marked as A11yBlocking, so it has to have higher priority than P2, because it will require an exception. We will also need an ETA on the fix.

**Comment by henrik-me** [comment]
@atiqhaqu : It is still not clear to me exactly what the issue is. Can you pls provide screenshots? If you are talking about the web dialog itself then we may not be the right team for this.
Also as this is open source you are welcome to contribute to the codebase. We will be happy to review your change and help getting the changes shipped.

**Comment by jmprieur** [comment]
Closing as @atiqhaqu has not replied for months.
